### DISCUSSION TITLE
Handle filter event in variables view

### DISCUSSION THREAD

**Discussion by suleymansahin** [microsoft/vscode-discussions#338] [Extension Development QnA] [answered]
Hello I like to handle event when I try to apply filter on variables view. When I try to search. When I look into the documents, unfortunately I couldnt find any information about it
![Error](https://user-images.githubusercontent.com/1921922/209154633-8874858c-2168-457e-8db8-2cfce2ff88b3.PNG)

**Comment by burkeholland** [comment]
@isidorn who might be the best person on the team to answer this? I looked through the docs myself and could not find anything - however I'm not sure how we refer to this component.

**Comment by isidorn** [comment, marked as answer]
This is not exposed to extensions. So currently not possible.
